.env

```diff
@@ -14,3 +14,5 @@ MAAT_API_OAUTH_URL=https://maat-cd-api-dev.auth.eu-west-2.amazoncognito.com
 MAAT_API_API_URL=https://maat-cd-api.dev.legalservices.gov.uk
 MAAT_API_CLIENT_ID=
 MAAT_API_CLIENT_SECRET=
+DATABASE_URL=postgres://postgres:postgres@postgres:5432/laa_court_data_adaptor_development
```

Comment:
Technically this wouldn't this need to go in the `.env.development.local` env file? Having it here means the build for the production image may pick it up.

Reply:
Good point, not sure if  the value of DATABASE_URL is replaced when deployed, but as you said it is enough to have it on the local env.